ShopAppTests/App/Account/ViewModels/Mocks/PersonalInfoViewModelMock.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  PersonalInfoViewModelMock.swift
+//  ShopAppTests
+//
+//  Created by Radyslav Krechet on 2/27/18.
+//  Copyright © 2018 RubyGarage. All rights reserved.
+//
+
+@testable import ShopApp
+
+class PersonalInfoViewModelMock: PersonalInfoViewModel {
+    func makeSaveChangesButtonEnabled() {
+        emailText.value = "u"
```

Comment:
Why you set emailText value when you can just emit event for saveChangesButtonEnabled observable?